app/Services/DoctorService.php

```diff
@@ -92,4 +92,25 @@ public function addWorkingDay($data)
         $workingDays = $this->repository->storeWorkingDay($data);
         return $workingDays;
     }
+
+    public function changePassword($data)
+    {
+        $doctor = $this->show(auth()->user()->id);
```

Comment:
No need to make a new query

just use the auth()->user()

`$doctor = auth()->user();`